### PR TITLE
javascript/gigasecond: add gigasecond automated comments

### PR DESCRIPTION
**Pull request by SleeplessByte**

### CODE REVIEW

File: automated-comments/javascript/gigasecond/prefer_number_comprehension.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Rewrite the literal `%{literal}` using `Math.pow` or `10 ** n` in order to make
```
Comment (ErikSchierboom) [inline automated-comments/javascript/gigasecond/prefer_number_comprehension.md]: Is this a disapproval, or an approval? If it is the latter, then maybe rephrase a bit to make it a suggestion instead of an order :)

Comment (SleeplessByte) [reply automated-comments/javascript/gigasecond/prefer_number_comprehension.md]: I think it currently is a `disapproval` 😓 😅

Comment (ErikSchierboom) [reply automated-comments/javascript/gigasecond/prefer_number_comprehension.md]: Alright! Then ignore my comment.
